projects/angular/src/forms/datepicker/_datepicker.clarity.scss

```
@@ -32,24 +33,42 @@
 }
 
 @mixin generate-calendar-focus-style() {
-  &:hover,
-  &:focus {
+  &:hover {
     background: datepicker-variables.$clr-calendar-btn-hover-focus-color;
   }
 
   &:focus {
     //Only because we have defined a background color for focus above
-    outline: 0;
+    outline: tokens.$cds-alias-object-interaction-outline;
+    outline-color: tokens.$cds-global-color-blue-700;
+    outline-offset: calc(-1 * tokens.$cds-global-space-2);
   }
 
   &:active {
     background: datepicker-variables.$clr-calendar-btn-active-color-bg;
   }
 }
 
+@mixin generate-current-date-style() {
+  &.is-today {
+    color: datepicker-variables.$clr-calendar-today-date-cell-color;
+    border: tokens.$cds-global-space-1 solid datepicker-variables.$clr-calendar-today-date-cell-border-color;
+    font-weight: datepicker-variables.$clr-calendar-today-date-cell-font-weight;
+  }
+}
+
+@mixin generate-selected-date-style() {
+  &.is-selected {
+    background: datepicker-variables.$clr-calendar-active-cell-background-color;
+    color: datepicker-variables.$clr-calendar-active-cell-color;
+    &:focus {
```

```suggestion
    color: datepicker-variables.$clr-calendar-active-cell-color;

    &:focus {
```

nit: Add blank line between blocks.